docs/webui.md

```diff
@@ -0,0 +1,240 @@
+---
+title: 'Gravitino web UI'
+slug: /webui
+keyword: webui
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+license: 'Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.'
+---
+
+import Image from '@theme/IdealImage'
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+This document primarily outlines how users can manage metadata within Gravitino using the web UI, the graphical interface accessible through a web browser as an alterative to writing code or using the REST interface.
+
+Currently, you can integrate [OAuth settings](security.md) to view, add, modify, and delete metalakes, create catalogs, and view catalogs, schemas, and tables, among other functions.
+
+:::caution
+More features are under development. For detailed capabilities, please refer to the [Manage metadata using Gravitino](manage-metadata-using-gravitino.md) document.
+:::
+
+[Build](how-to-build#quick-start) and [deploy](getting-started#getting-started-locally) the Gravitino Web UI and open it in a browser at [http://localhost:8090](http://localhost:8090).
+
+## Initial page
+
+The web UI homepage displayed in Gravitino depends on the configuration parameter for OAuth mode, see the details in [Security](security.md).
+
+Set parameter for `gravitino.authenticator`, [`simple`](#simple-mode) or [`oauth`](#oauth-mode).
+
+:::tip
+After changing the configuration, make sure to restart the Gravitino server.
+
+`<path-to-gravitino>/bin/gravitino.sh restart`
+:::
+
+### Simple mode
+
+```
+gravitino.authenticator = simple
+```
+
+Set the configuration parameter for `gravitino.authenticator` to `simple`, the web UI directly displays the homepage (Metalakes).
+
+![webui-metalakes-simple](assets/webui/metalakes-simple.png)
+
+At the top-right, displays the current version.
```

Comment:
At the top-right, the UI displays the current Gravitino version.

Reply:
DONE